src/main.js

```diff
@@ -3,18 +3,26 @@ var title = document.querySelector('h2');
 var desc1 = document.querySelector('.tagline-1');
 var desc2 = document.querySelector('.tagline-2');
 var image = document.querySelector('.cover-image');
-var btn = document.querySelector('.random-cover-button');
+var priceTag = document.querySelector('.price-tag');
+var tagLine = document.querySelector('.tagline');
+var btnRandomCover = document.querySelector('.random-cover-button');
+var btnMakeCover = document.querySelector('.make-new-button')
+var homePage =[title, desc1, desc2, image, priceTag, tagLine];
 // We've provided a few variables below
 var savedCovers = [
   new Cover("http://3.bp.blogspot.com/-iE4p9grvfpQ/VSfZT0vH2UI/AAAAAAAANq8/wwQZssi-V5g/s1600/Do%2BNot%2BForsake%2BMe%2B-%2BImage.jpg", "Sunsets and Sorrows", "sunsets", "sorrows")
 ];
 var currentCover;
 
 // Add your event listeners here 👇
```

Comment:
See if we can use 'load' event on window to call function , can hopefully refactor by doing this